harmonica/_euler_deconvolution.py

```diff
@@ -0,0 +1,87 @@
+# Copyright (c) 2018 The Harmonica Developers.
+# Distributed under the terms of the BSD 3-Clause License.
+# SPDX-License-Identifier: BSD-3-Clause
+#
+# This code is part of the Fatiando a Terra project (https://www.fatiando.org)
+#
+import numpy as np
+import scipy as sp
+
+
+class EulerDeconvolution:
+    r"""
+    Euler deconvolution for estimating source location and base level from
+    potential field measurements.
```

Comment:
```suggestion
    Estimate source location and base level using Euler Deconvolution
```

Harmonica is only about potential field data so this isn't needed. The first part of the docstring should be a single line.